doc/Makefile

```diff
@@ -144,6 +144,10 @@ doctest:
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
+fill_doc_check:
+	mkdir -p tmp
+	grep --include "*.html" -rn "_build/html/" -e "%(" > tmp/doc_check.txt
```

Comment:
how does one escape `(` in makefile?

Reply:
Try  "\\%("

Reply:
> Try "\%("

If that doesn't work then let's leave it to your original implementation. Thanks for trying

Reply:
Oops wait

Reply:
the suspense is unbearable. 🤗

Reply:
So it seems this `"\%("` and this `"\\%("` should work. It's kind of what I got from here https://www.cmcrossroads.com/article/gnu-make-escaping-walk-wild-side with a little trial and error

Reply:
did not seem to work locally
let's see what CI says

Reply:
Hmm what did I do 😅 . Ok NVM

Reply:
moved the bash script with its friends in the build_tools folder. 🫂

Reply:
I'm sorry this is bugging me and no need to change anything but I know what's happening! grep gives exit status 1 when no match is found and error is status >1. Makefile sees exit status is not 0 and errors out. Doesn't even seem like you need to escape anything. Initially I tested on a case where it could find a match and it worked but since you fixed all of them it wasn't finding anything

Reply:
you have more patience than I do. 🙏🏾